### ISSUE TITLE
Deprecated personal access token

### ISSUE THREAD

**Issue by dkulp2** [r-lib/remotes#471]
I received an automated notice from GitHub enterprise today warning me that passing a personal access token as a query parameter is deprecated. They say they'll continue to bug me every three days until I fix it. 

I'm using `install_github(repo, auth_token=XXX, ...)`

Is there a development plan to address this?

> ---------- Forwarded message ----------
> From: GitHub <noreply@github.com>
> Date: Feb 3, 2020, 5:35 PM -0500
> To: David Kulp 
> Subject: [GitHub API] Deprecation notice for authentication via URL query parameters
> 
> Hi @dkulp2,
> 
> On February 3rd, 2020 at 22:35 (UTC) your personal access token using libcurl/7.52.1 was used as part of a query parameter to access an endpoint through the GitHub API:
> 
> https://api.github.com/repositories/...
> 
> Please use the Authorization HTTP header instead, as using the `access_token` query parameter is deprecated.
> 
> Depending on your API usage, we'll be sending you this email reminder once every 3 days for each token and User-Agent used in API calls made on your behalf.
> Just one URL that was accessed with a token and User-Agent combination will be listed in the email reminder, not all.
> 
> Visit https://developer.github.com/changes/2019-11-05-deprecated-passwords-and-authorizations-api/#authenticating-using-query-parameters for more information.
> 
> Thanks,
> The GitHub Team

**Comment by jimhester** [comment]
remotes no longer uses the `access_token` query parameter after https://github.com/r-lib/remotes/commit/0c980e2cfbbb9cbdd6d8b515069fad30140b008e, which was released on CRAN with version 2.1.0 of remotes, released in June of 2019.

**Comment by dkulp2** [comment]
For the record, I've discovered that my problem was because I am using  the rocker docker container and `install2.r`. Rocker includes the handy `install2.r` script, which is from the `littler` package. `install2.r` is useful to use in a dockerfile because it will cause an error on a failed installation. I found that [rocker's dockerfile](https://hub.docker.com/r/rocker/r-ver/dockerfile) sets the `repos` option to an MRAN snapshot by writing to /usr/local/lib/R/etc/Rprofile.site. In my case, the snapshot is from 2019-06-13, presumably before v 2.1.0 of remotes.
Using `RUN install2.r -e -r https://cran.rstudio.com/ devtools` in your dockerfile will install the latest version of remotes.
Re-pulling rocker would probably also do the trick!
